site/src/components/pages/docs/search/search-suggested-links.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, buttonClasses, Typography, useTheme } from "@mui/material";
+import React from "react";
+
+import { Button } from "../../../button";
+import { SpecificationIcon } from "../../../icons";
+import { LinkButton } from "../../../link-button";
+
+interface SearchSuggestedLinksProps {
+  setSearchText?: (text: string) => void;
+  closeModal?: () => void;
+}
+
+const SUGGESTED_LINKS = [
+  "Introduction to the Block Protocol",
+  "Quick start guide to developing blocks",
+  "A guide for embedding applications",
```

Comment:
```suggestion
  "Embed blocks within your site or app",
```
We haven't published our actual guide for embedding applications yet, and we only explain what the 'EA' term means on certain pages. I think this is a clearer CTA that relates to our current public content better.